src/Sulu/Bundle/SecurityBundle/Resources/config/doctrine/BaseUser.orm.xml

```diff
@@ -24,7 +24,13 @@
         </field>
         <field name="lastLogin" column="lastLogin" type="datetime" nullable="true" />
         <field name="confirmationKey" column="confirmationKey" type="string" length="128" nullable="true" />
-        <field name="passwordResetToken" column="passwordResetToken" type="string" length="256" nullable="true" />
+        <field name="passwordResetToken" column="passwordResetToken" type="string" length="80" nullable="true" unique="true" />
+        <field name="tokenExpiresAt" column="tokenExpiresAt" type="datetime" nullable="true" />
+        <field name="tokenEmailsSent" column="tokenEmailsSent" type="integer" nullable="true">
```

Comment:
Would prefix the previous two fields also with `passwordReset`, just in case there will be more tokens in the future.